test/suite/suite_after_each_hook_title.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Per Eckerdal
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+afterEach(function() {
+  console.log(this.currentTest.title);
+  console.log(this.currentTest.fullTitle());
```

Comment:
wow, that is a weird api. One being a function and one a variable. But it seems like that's what mocha does so all good